connector-drawer/src/main/java/com/redhat/cloud/notifications/connector/drawer/DrawerRouteBuilder.java

```diff
@@ -13,15 +16,37 @@
 public class DrawerRouteBuilder extends EngineToConnectorRouteBuilder {
 
     @Inject
-    ConnectorConfig drawerConnectorConfig;
+    DrawerConnectorConfig drawerConnectorConfig;
+
+    @Inject
+    RecipientsResolverPreparer recipientsResolverPreparer;
+
+    @Inject
+    RecipientsResolverResponseProcessor recipientsResolverResponseProcessor;
+
+    @Inject
+    DrawerPayloadBuilder drawerPayloadBuilder;
+
+    public static final String CONNECTOR_TO_DRAWER = "connector-to-drawer";
 
     @Override
     public void configureRoute() {
         from(seda(ENGINE_TO_CONNECTOR))
             .routeId(drawerConnectorConfig.getConnectorName())
+            .process(recipientsResolverPreparer)
+            .to(drawerConnectorConfig.getRecipientsResolverServiceURL() + "/internal/recipients-resolver")
+            .process(recipientsResolverResponseProcessor)
+            .split(simpleF("${exchangeProperty.%s}", ExchangeProperty.RESOLVED_RECIPIENT_LIST))
+                .process(drawerPayloadBuilder)
+                .to(log(getClass().getName()).level("INFO").showHeaders(true).showBody(true))
+                .to(direct(CONNECTOR_TO_DRAWER))
```

Comment:
I don't really have a preference, but this could be:
```suggestion
                .to(kafka(drawerConnectorConfig.getOutgoingDrawerTopic()))
```

Reply:
After many tries, I conclude we need to define a dedicated route to be able to check sent data on unit tests.